tools/vscode/ddlog-theme/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Change Log
```

Comment:
We do not really maintain change logs anymore, these are maintained by git automatically, so you can probably delete this file.